.golangci.yaml

```diff
@@ -20,20 +21,18 @@ linters:
   - asciicheck
   - containedctx
   - decorder
-  - depguard
-  - dogsled
   - errcheck
   - errorlint
   - exportloopref
   - goconst
   - gocritic
-  - gocyclo
   - godot
   - gofmt
   - goimports
   - gosec
   - gosimple
   - govet
+  - gocyclo
```

Comment:
Nit: le'ts add it in alphabetical order.

Reply:
done